<launch>
  <node pkg="nodelet" type="nodelet" name="mynteye"
    args="standalone mynt_wrapper/MYNTWrapperNodelet" output="screen">

    <param name="device_name"   type="int"  value="1" />

    <!-- 0: RATE_50_FPS_500_HZ, 1: RATE_25_FPS_500_HZ, 2: RATE_10_FPS_200_HZ -->
    <param name="camera_rate"   type="int"  value="1" />

    <param name="left_raw_topic"        value="left/image_raw" />
    <param name="left_cam_info_topic"   value="left/camera_info" />
    <param name="right_raw_topic"       value="right/image_raw" />
    <param name="right_cam_info_topic"  value="right/camera_info" />
    <param name="imu_topic"             value="imu" />
  </node>
</launch>
